### PR TITLE
RoutineExerciseView First Pull Request

### PR DESCRIPTION
**Pull request by cheniwei**
# RoutineExerciseView First Pull Request
## RoutineExerciseView
1. Add new exercise by clicking "**+ Add Exercise**" to today's routine list. -> Link to AddExerciseButton view
2. Click "**Finish**" button to back to MainView.
4. Click "**Edit**" button to delete / move exercises in routine list.
Screenshot 2023-04-18 at 3 35 36 PM

## AddExerciseButton
1. Wheel picker for users to choose an existed exercise and click "**Done**" to add it to the routine list.
2. If there is no exercise the user wants, click "**New**" to add new exercise. -> Link to AddNewExercise
Screenshot 2023-04-18 at 3 36 10 PM

## AddNewExercise:
1. TextField to type new exercise.
2. Click "**SAVE**" button to save the new exercise.
Screenshot 2023-04-18 at 3 36 32 PM

### CODE REVIEW

**Review by alex308248 — COMMENTED** [review]
It looks awesome!! 
The full process of uploading your code looks like the following: 
1. Commit your changes on your local machine and make sure it's working well
2. Pull the latest version of branch you're doing pr (eg. version1)
3. Merge version1 and your branch in your local machine and make sure there are no conflict.
4. Push your branch to github repo
5. Write your pull request and assign reviewers

An Example of the commands:
```
git checkout version1
git pull origin version1
git checkout cfork
git merge version1
git push
```

File: GymBro/Views/RoutineViews/AddExerciseButton.swift [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+//  This view creates a pop-up sheet including a wheel picker for user to
+//  choose which exercise to add to the routine list of day.
+//  By clicking "New" button, the user can add an new exercise that does not
+//  exist to the wheel picker.
+
+import SwiftUI
+
+struct AddExerciseButton: View {
+    
+    // MARK: PROPERTY
```
Comment (YLee2121) [inline GymBro/Views/RoutineViews/AddExerciseButton.swift:17]: MARK guideline: 

Two blank lines above MARK and one blank line below MARK.

File: GymBro/Views/RoutineViews/AddExerciseButton.swift [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                selection: $selection,
+                content: {
+                    ForEach(exercise, id: \.self, content: {
+                        Text($0)
+                    })
+                })
+            .pickerStyle(.wheel)
+                
+            HStack {
+                // Add a new exercise to the routine list
```
Comment (YLee2121) [inline GymBro/Views/RoutineViews/AddExerciseButton.swift:44]: Comment guideline:

It needs a blank line above the comment.

File: GymBro/Views/RoutineViews/AddExerciseButton.swift [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+//  Created by Claire on 4/11/23.
+//
+//  This view creates a pop-up sheet including a wheel picker for user to
+//  choose which exercise to add to the routine list of day.
+//  By clicking "New" button, the user can add an new exercise that does not
+//  exist to the wheel picker.
+
+import SwiftUI
+
+struct AddExerciseButton: View {
```
Comment (YLee2121) [inline GymBro/Views/RoutineViews/AddExerciseButton.swift:14]: Documentation guideline:

If it's possible, please add some simple documentation. 
Check the "🎮Xcode Document" section in the GuideLines.md files.

**Review by YLee2121 — CHANGES_REQUESTED** [review]
In general, it's tidy and clean. 
Good job on the first PR.

If it's convenient, could you please add screenshots of the iPhone Simulator to the PR report? It could help us imagine the code more easily.
Please check the comment about the documentation and try to solve them. (all the script files)

**Comment by alex308248** [comment]
This PR should be merging to `version1` but not main. Don't forget to change it!

**Review by YLee2121 — APPROVED** [review]
Approved!

@alex308248 after you approve the PR, please press `Squash and merge`

**Review by alex308248 — APPROVED** [review]
Approved!

**Pull request merged**